src/label-for-moment/label-for-moment.js

```diff
@@ -0,0 +1,26 @@
+import _ from 'lodash';
+
+export default function labelForMoment(date) {
```

Comment:
just realized we're not showing time in the best format, sorry

We should always show time, rather than this time ago

`10:32am` (today)
`4/26 - 10:32am` (this year)
`4/26/95 - 10:32am` (otherwise)

You shouldn't be using Date - this function should be about 6 lines - just some if statements like so (please just copy and paste this and leave the comments and fill in the blanks:

labelForMoment(momentInstance) {
  var format = ''; //standard long form - m/d/y 00:00am

  if () {
    format = ''; //today - 00:00am
  }
  else if () {
    format = ''; //this year - m/d 00:00am
  }

  return momentInstance.format(format);
}